packages/common/src/utils/ConstantsUtil.ts

```diff
@@ -2,5 +2,6 @@ export const ConstantsUtil = {
   WC_NAME_SUFFIX: '.wcn.id',
   BLOCKCHAIN_API_RPC_URL: 'https://rpc.walletconnect.org',
   PULSE_API_URL: 'https://pulse.walletconnect.org',
-  W3M_API_URL: 'https://api.web3modal.org'
+  W3M_API_URL: 'https://api.web3modal.org',
+  APPKIT_AUTH_API_URL: 'https://api-web3modal-staging.walletconnect-v1-bridge.workers.dev'
```

Comment:
Is this expected to be changed? Do we some changes to be deployed to w3m API?

Reply:
No this is temporary until the Auth API routes are live in prod